### DISCUSSION TITLE
Deployment

### DISCUSSION THREAD

**Discussion by DylanKarimagoko** [iptv-org/community#575] [Q&A]
How would you deploy the TV guide to vercel where in there src file eg index, server etc

**Comment by Animenosekai** [comment]
You need to generate the static XMLTV files and update it frequently.

To update it, you need to run the scripts again, commit to the repository and let Vercel or other hosting platforms deploy it again.

If you want an example, here is what I did : [Animenosekai/japanterebi-xmltv](https://github.com/Animenosekai/japanterebi-xmltv)

You might want to look at the GitHub Action where all the steps are shown : [Animenosekai/japanterebi-xmltv/.github/workflows/update.yaml](https://github.com/Animenosekai/japanterebi-xmltv/blob/main/.github/workflows/update.yaml)
